H2TauTau/python/heppy/analyzers/DiLeptonAnalyzer.py

```diff
@@ -34,6 +54,19 @@ def pt(self):
     def sumPt(self):
         return self.leg1().pt() + self.leg2().pt()
 
+    @staticmethod
+    def calcMT(cand1, cand2):
+        pt = cand1.pt() + cand2.pt()
+        px = cand1.px() + cand2.px()
+        py = cand1.py() + cand2.py()
+        try:
+            return math.sqrt(pt*pt - px*px - py*py)
+        except ValueError:
+            print 'Funny rounding issue', pt, px, py
```

Comment:
ne devrait-on pas retourner une valeur non physique, genre -1 ? 
est-ce que ca ne peut se produire que lorsqu'on a une erreur de rounding? 
le probleme n'est-il pas plutot que mt^2 est = 0 et legerement negatif? 
si oui, modifier le message en: 
'mT2 very close to 0 and negative due to rounding'